versioning.md

```diff
@@ -30,6 +28,44 @@ Whether a new version is a patch release, a minor release or a major release sho
 following Semantic Versioning. https://semver.org/
 
 
+## Changelog
+
+All projects should keep a changelog. A file called CHANGELOG.md should be in the root of the repository.
```

Comment:
This feels like one of those things that is accidentally more prescriptive than intended (unless you all agreed to change all our stuff this morning!). 

What's meant by 'project'? Is it 'repo'?

Is it the case that "if I see a repo without a changelog I should add one as a matter of priority', or 'if I create a new repo, I should add a changelog by default'? Or, something else?

Reply:
Good point, it should be something like "all projects versioned in this way ....". Will amend.